[Bug]: opusic-sys build fail

### Expected Behavior

build

### Actual Behavior

fail

### Steps to Reproduce

cd new-client
namui start wasm32-wasi-web

### Other Info

```text
error: failed to run custom build command for `opusic-sys v0.5.1`

Caused by:
  process didn't exit successfully: `CARGO=/home/bigfood/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/bin/cargo CARGO_CFG_PANIC=abort CARGO_CFG_TARGET_ABI='' CARGO_CFG_TARGET_ARCH=wasm32 CARGO_CFG_TARGET_ENDIAN=little CARGO_CFG_TARGET_ENV=p1 CARGO_CFG_TARGET_FAMILY=wasm CARGO_CFG_TARGET_FEATURE=bulk-memory,mutable-globals,sign-ext,simd128 CARGO_CFG_TARGET_HAS_ATOMIC=16,32,64,8,ptr CARGO_CFG_TARGET_OS=wasi CARGO_CFG_TARGET_POINTER_WIDTH=32 CARGO_CFG_TARGET_VENDOR=unknown CARGO_ENCODED_RUSTFLAGS='-Ctarget-feature=-crt-static-L/home/bigfood/programming/namseent/namui/namui-cli/wasi-sdk/share/wasi-sysroot/lib/wasm32-wasip1-threads-L/home/bigfood/programming/namseent/namui/namui-cli/wasi-sdk/lib/clang/18/lib/wasip1-Clink-arg=--initial-memory=8388608-Clink-arg=--max-memory=4294967296-Clink-arg=--stack-first-Clink-arg=--export=__heap_base-Clink-arg=--export=__data_end-Clink-arg=--export=malloc-Clink-arg=--export=free-Ctarget-feature=+simd128' CARGO_MANIFEST_DIR=/home/bigfood/.cargo/registry/src/index.crates.io-6f17d22bba15001f/opusic-sys-0.5.1 CARGO_PKG_AUTHORS='Douman <douman@gmx.se>' CARGO_PKG_DESCRIPTION='Bindings to libopus' CARGO_PKG_HOMEPAGE='' CARGO_PKG_LICENSE=BSL-1.0 CARGO_PKG_LICENSE_FILE='' CARGO_PKG_NAME=opusic-sys CARGO_PKG_README=README.md CARGO_PKG_REPOSITORY='https://github.com/DoumanAsh/opusic-sys' CARGO_PKG_RUST_VERSION='' CARGO_PKG_VERSION=0.5.1 CARGO_PKG_VERSION_MAJOR=0 CARGO_PKG_VERSION_MINOR=5 CARGO_PKG_VERSION_PATCH=1 CARGO_PKG_VERSION_PRE='' CARGO_TARGET_WASM32_WASIP1_THREADS_LINKER=/home/bigfood/programming/namseent/namui/namui-cli/wasi-sdk/bin/wasm-ld CC=/home/bigfood/programming/namseent/namui/namui-cli/wasi-sdk/bin/clang CLANGCC=/home/bigfood/programming/namseent/namui/namui-cli/wasi-sdk/bin/clang CLANGCXX=/home/bigfood/programming/namseent/namui/namui-cli/wasi-sdk/bin/clang++ CLANG_PATH=/home/bigfood/programming/namseent/namui/namui-cli/wasi-sdk/bin/clang CXX=/home/bigfood/programming/namseent/namui/namui-cli/wasi-sdk/bin/clang++ DEBUG=true EMSDK_SYSTEM_INCLUDE=/home/bigfood/programming/namseent/namui/namui-cli/emscripten/system/include HOST=x86_64-unknown-linux-gnu LD_LIBRARY_PATH='/home/bigfood/programming/namseent/luda-editor/new-client/target/namui/target/debug/deps:/home/bigfood/programming/namseent/luda-editor/new-client/target/namui/target/debug:/home/bigfood/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib:/home/bigfood/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib' NUM_JOBS=8 OPT_LEVEL=1 OUT_DIR=/home/bigfood/programming/namseent/luda-editor/new-client/target/namui/target/wasm32-wasip1-threads/debug/build/opusic-sys-b467a85a181b73e2/out PROFILE=debug RUSTC=/home/bigfood/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/bin/rustc RUSTDOC=/home/bigfood/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/bin/rustdoc TARGET=wasm32-wasip1-threads WASI_SDK=/home/bigfood/programming/namseent/namui/namui-cli/wasi-sdk WASI_SYSROOT=/home/bigfood/programming/namseent/namui/namui-cli/wasi-sdk/share/wasi-sysroot /home/bigfood/programming/namseent/luda-editor/new-client/target/namui/target/debug/build/opusic-sys-a3448b4eba6ffea2/build-script-build` (exit status: 101)
  --- stdout
  cargo:rerun-if-env-changed=OPUS_LIB_DIR
  cargo:rerun-if-env-changed=ANDROID_NDK_HOME
  1.10.1
  CMAKE_TOOLCHAIN_FILE_wasm32-wasip1-threads = None
  CMAKE_TOOLCHAIN_FILE_wasm32_wasip1_threads = None
  TARGET_CMAKE_TOOLCHAIN_FILE = None
  CMAKE_TOOLCHAIN_FILE = None
  CMAKE_PREFIX_PATH_wasm32-wasip1-threads = None
  CMAKE_PREFIX_PATH_wasm32_wasip1_threads = None
  TARGET_CMAKE_PREFIX_PATH = None
  CMAKE_PREFIX_PATH = None
  CMAKE_wasm32-wasip1-threads = None
  CMAKE_wasm32_wasip1_threads = None
  TARGET_CMAKE = None
  CMAKE = None
  running: cd "/home/bigfood/programming/namseent/luda-editor/new-client/target/namui/target/wasm32-wasip1-threads/debug/build/opusic-sys-b467a85a181b73e2/out/build" && CMAKE_PREFIX_PATH="" "cmake" "/home/bigfood/.cargo/registry/src/index.crates.io-6f17d22bba15001f/opusic-sys-0.5.1/opus" "-G" "Ninja" "-DOPUS_INSTALL_PKG_CONFIG_MODULE=OFF" "-DOPUS_INSTALL_CMAKE_CONFIG_MODULE=OFF" "-DCMAKE_INSTALL_BINDIR=bin" "-DCMAKE_INSTALL_MANDIR=man" "-DCMAKE_INSTALL_INCLUDEDIR=include" "-DCMAKE_INSTALL_OLDINCLUDEDIR=include" "-DCMAKE_INSTALL_LIBDIR=lib" "-DCMAKE_SYSTEM_NAME=wasi" "-DCMAKE_SYSTEM_PROCESSOR=wasm32" "-DCMAKE_INSTALL_PREFIX=/home/bigfood/programming/namseent/luda-editor/new-client/target/namui/target/wasm32-wasip1-threads/debug/build/opusic-sys-b467a85a181b73e2/out" "-DCMAKE_C_FLAGS= -ffunction-sections -fdata-sections -fno-exceptions --sysroot=/home/bigfood/programming/namseent/namui/namui-cli/wasi-sdk/share/wasi-sysroot -pthread --target=wasm32-wasi" "-DCMAKE_C_COMPILER=/home/bigfood/programming/namseent/namui/namui-cli/wasi-sdk/bin/clang" "-DCMAKE_CXX_FLAGS= -ffunction-sections -fdata-sections -fno-exceptions --sysroot=/home/bigfood/programming/namseent/namui/namui-cli/wasi-sdk/share/wasi-sysroot -pthread --target=wasm32-wasi" "-DCMAKE_CXX_COMPILER=/home/bigfood/programming/namseent/namui/namui-cli/wasi-sdk/bin/clang++" "-DCMAKE_ASM_FLAGS= -ffunction-sections -fdata-sections -fno-exceptions --sysroot=/home/bigfood/programming/namseent/namui/namui-cli/wasi-sdk/share/wasi-sysroot -pthread --target=wasm32-wasi" "-DCMAKE_ASM_COMPILER=/home/bigfood/programming/namseent/namui/namui-cli/wasi-sdk/bin/clang" "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
  -- Opus package version from package_version file: 1.5.2
  -- Opus project version: 1.5.2
  -- The C compiler identification is Clang 18.1.2
  -- Detecting C compiler ABI info
  -- Detecting C compiler ABI info - failed
  -- Check for working C compiler: /home/bigfood/programming/namseent/namui/namui-cli/wasi-sdk/bin/clang
  -- Check for working C compiler: /home/bigfood/programming/namseent/namui/namui-cli/wasi-sdk/bin/clang - broken
  -- Configuring incomplete, errors occurred!
  See also "/home/bigfood/programming/namseent/luda-editor/new-client/target/namui/target/wasm32-wasip1-threads/debug/build/opusic-sys-b467a85a181b73e2/out/build/CMakeFiles/CMakeOutput.log".
  See also "/home/bigfood/programming/namseent/luda-editor/new-client/target/namui/target/wasm32-wasip1-threads/debug/build/opusic-sys-b467a85a181b73e2/out/build/CMakeFiles/CMakeError.log".

  --- stderr
  System is unknown to cmake, create:
  Platform/wasi to use this system, please post your config file on discourse.cmake.org so it can be added to cmake
  Your CMakeCache.txt file was copied to CopyOfCMakeCache.txt. Please post that file on discourse.cmake.org.
  System is unknown to cmake, create:
  Platform/wasi to use this system, please post your config file on discourse.cmake.org so it can be added to cmake
  System is unknown to cmake, create:
  Platform/wasi to use this system, please post your config file on discourse.cmake.org so it can be added to cmake
  CMake Error at /usr/share/cmake-3.22/Modules/CMakeTestCCompiler.cmake:69 (message):
    The C compiler

      "/home/bigfood/programming/namseent/namui/namui-cli/wasi-sdk/bin/clang"

    is not able to compile a simple test program.

    It fails with the following output:

      Change Dir: /home/bigfood/programming/namseent/luda-editor/new-client/target/namui/target/wasm32-wasip1-threads/debug/build/opusic-sys-b467a85a181b73e2/out/build/CMakeFiles/CMakeTmp
      
      Run Build Command(s):/usr/bin/ninja cmTC_c5846 && [1/2] Building C object CMakeFiles/cmTC_c5846.dir/testCCompiler.c.obj
      [2/2] Linking C executable cmTC_c5846
      FAILED: cmTC_c5846 
      : && /home/bigfood/programming/namseent/namui/namui-cli/wasi-sdk/bin/clang -ffunction-sections -fdata-sections -fno-exceptions --sysroot=/home/bigfood/programming/namseent/namui/namui-cli/wasi-sdk/share/wasi-sysroot -pthread --target=wasm32-wasi  CMakeFiles/cmTC_c5846.dir/testCCompiler.c.obj -o cmTC_c5846   && :
      wasm-ld: error: --shared-memory is disallowed by errno.o because it was not compiled with 'atomics' or 'bulk-memory' features.
      clang: error: linker command failed with exit code 1 (use -v to see invocation)
      ninja: build stopped: subcommand failed.
      
      

    

    CMake will not be able to correctly generate this project.
  Call Stack (most recent call first):
    CMakeLists.txt:7 (project)


  thread 'main' panicked at /home/bigfood/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cmake-0.1.51/src/lib.rs:1100:5:

  command did not execute successfully, got: exit status: 1

  build script failed, must exit now
  stack backtrace:
     0: rust_begin_unwind
               at /rustc/f6e511eec7342f59a25f7c0534f1dbea00d01b14/library/std/src/panicking.rs:662:5
     1: core::panicking::panic_fmt
               at /rustc/f6e511eec7342f59a25f7c0534f1dbea00d01b14/library/core/src/panicking.rs:74:14
     2: cmake::fail
               at /home/bigfood/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cmake-0.1.51/src/lib.rs:1100:5
     3: cmake::run
               at /home/bigfood/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cmake-0.1.51/src/lib.rs:1078:9
     4: cmake::Config::build
               at /home/bigfood/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cmake-0.1.51/src/lib.rs:814:13
     5: build_script_build::build
               at ./build.rs:114:23
     6: build_script_build::run
               at ./build.rs:132:9
     7: build_script_build::main
               at ./build.rs:137:5
     8: core::ops::function::FnOnce::call_once
               at /rustc/f6e511eec7342f59a25f7c0534f1dbea00d01b14/library/core/src/ops/function.rs:250:5
  note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
warning: build failed, waiting for other jobs to finish...

```

--- a/namui/namui/Cargo.toml
+++ b/namui/namui/Cargo.toml
@@ -47,7 +47,7 @@ hyper = "1.4.1"
 http-body-util = "0.1.2"
 ogg = { version = "0.9.1" }
 pin-project = "1"
-opusic-sys = "0.5.1"
+opusic-sys = "0.5.2"
 
 [target.'cfg(not(target_os="wasi"))'.dependencies]
 hyper-rustls = { version = "0.27.2", default-features = false, features = [

